lib/rubocop/config_loader.rb

```diff
@@ -91,7 +91,9 @@ def configuration_from_file(config_file)
         else
           add_excludes_from_files(config, config_file)
         end
-        merge_with_default(config, config_file)
+        merge_with_default(config, config_file).tap do |merged_config|
```

Comment:
`merge_with_default` does not mutate first argument, but rather returns merged config, so I was between `tap` and using a temporary local variable.